tests/api/task1.test.js

```diff
@@ -0,0 +1,36 @@
+import {test, expect} from '@playwright/test';
+
+test.describe("API testing", () => {
+  const baseUrl = "http://localhost:3030/"
+  test.beforeAll(async ({request}) => {
```

Comment:
I would recommend changing `beforeAll` to `beforeEach`. If we clean the database only once at the beginning then there is a chance the first test can affect the subsequent tests.